### ISSUE TITLE
Support for Community Service as punishment

### ISSUE THREAD

**Issue by dexslab** [SnailyCAD/snaily-cadv4#1659]
### Describe the feature

When LEO give jail or warning or fine also add the ability to put them on probation to do community service. Just needs to accept a string of what probation entails. This is for use with prison scripts that have community service like rcore thanks

**Comment by Dev-CasperTheGhost** [comment]
This would be a field called "Community Service" (`communityService?: string`)? @dexslab

**Comment by dexslab** [comment]
Yes this would be just that a string for community service arrest record and can be used in conjunction with fine like you can with jail time
